runtime/vm/exceptiondescribe.c

```diff
@@ -234,12 +255,29 @@ findJ9ClassForROMClass(J9VMThread *vmThread, J9ROMClass *romClass, J9ClassLoader
 {
```

Comment:
Anon classes are not in any class loader hahstable. This function can directly return null if `_J9ROMCLASS_J9MODIFIER_IS_SET(romClass, J9AccClassAnonClass)` is true.